### DISCUSSION TITLE
What it happens if I truncate the stored_events table?

### DISCUSSION THREAD

**Discussion by raphaelcangucu** [spatie/laravel-event-sourcing#470] [Q&A]
It will break or anything?

My table is getting huge...

**Comment by inmanturbo** [comment]
You won't be able to replay the events that get deleted. You could always back them up first. You could even make a ~projector~ reactor that copies your new events to the backup table in case you ever needed to replay from it🤷

**Reply by raphaelcangucu** [reply]
That's the only problem? Replay the events?

I guess that the event-sourcing was wrongly implemented on our system ... Because we have all the "events" at the dynamo already.

Ok thanks a lot.
